src/main/cpp/SDLPlayData.cpp

```diff
@@ -747,8 +745,9 @@ void SDLPlayData::video_refresh(VideoState *is, double *remaining_time) {
 			else if (is->get_audio_st())
 				av_diff = is->get_master_clock() - is->get_pAudclk()->get_clock();
 			av_log(NULL, AV_LOG_INFO,
-				"%7.2f %s:%7.3f fd=%4d aq=%5dKB vq=%5dKB sq=%5dB f=%f /%f   \r",
+				"%7.2f at %dX %s:%7.3f fd=%4d aq=%5dKB vq=%5dKB sq=%5dB f=%f /%f   \r",
```

Comment:
When testing this PR I did not see the dX updated! (But the speed changed)

Reply:
When enabling the filters, the clock speed et set to 0 even changing the clock speed during playback; the master clock will remain 0.